src/Customerio/Request.php

```diff
@@ -161,6 +161,34 @@ public function anonymousEvent($name, $data)
         return new Response($response->getStatusCode(), $response->getReasonPhrase());
     }
 
+    public function addToSegment($segment, $users = [])
```

Comment:
This class uses phpDoc, to be consistent, you should use it for this method too.

Reply:
All done :+1: